spec/mock/expect-mock.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = (thing) => {
+
```

Comment:
Got this changed, linter complained about missing JsDoc as well once I got that fixed.